components/Cards.js

```diff
@@ -19,4 +19,82 @@
 //
 // Add a listener for click events so that when a user clicks on a card, the headline of the article is logged to the console.
 //
+
+const { default: Axios } = require("axios");
+
 // Use your function to create a card for each of the articles, and append each card to the DOM.
+
+const headlineEntryPoint = document.querySelector('div.cards-container')
+
+Axios.get('https://lambda-times-api.herokuapp.com/articles')
+.then(({data}) => {
+    // console.log(data)
+    let articlesJS = data.articles['javascript']
```

Comment:
You can loop through each of the topics (javascript, bootstrap, etc) using Object.entries

Here's a link explaining more. [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries)

This isn't something you'd be expected to know but it's a good coding style adjustment.

You can also read about how to use a for loop to accomplish the same thing [https://stackoverflow.com/questions/684672/how-do-i-loop-through-or-enumerate-a-javascript-object#:~:text=You%20can%20use%20the%20for,t%20come%20from%20the%20prototype.&text=Using%20the%20new%20Object](https://stackoverflow.com/questions/684672/how-do-i-loop-through-or-enumerate-a-javascript-object#:~:text=You%20can%20use%20the%20for,t%20come%20from%20the%20prototype.&text=Using%20the%20new%20Object).